exchange/price_granularity.go

```diff
@@ -15,21 +16,27 @@ func GetCpmStringValue(cpm float64, config openrtb_ext.PriceGranularity) (string
 	bucketMax := 0.0
```

Comment:
While we're here, consider renaming the method to `GetPriceBucket` for improved clarity.